packages/front-end/components/Layout/Layout.tsx

```diff
@@ -190,6 +190,7 @@ const navlinks: SidebarLinkProps[] = [
         href: "/integrations/slack",
         path: /^integrations\/slack/,
         feature: "slack-integration",
+        permissions: ["manageIntegrations"],
```

Comment:
This is the permission the Slack integration page `/integrations/slack/index.js` uses to determine whether or not the user can see the Slack management page.